src/shogun/machine/gp/SingleFITCLaplacianInferenceMethod.cpp

```diff
@@ -80,7 +82,7 @@ class CFITCPsiLine : public func_base
 		(*dlp)=lik->get_log_probability_derivative_f(lab, (*f), 1);
 
 		(*W)=lik->get_log_probability_derivative_f(lab, (*f), 2);
-		W->scale(-1.0);
+		linalg::scale<linalg::Backend::NATIVE>(*W, -1.0);
```

Comment:
Not too relevant. But I am not sure why `SGVector<T>*` was used here. `SGVector<T>` is supposed to be used by value (mostly). Maybe @yorkerlin can comment since he has written the code.